frontend/src/pages/projects/components/PVSizeField.tsx

```diff
@@ -17,41 +26,42 @@ const PVSizeField: React.FC<PVSizeFieldProps> = ({ fieldID, size, setSize, curre
     setSize(Math.max(size + step, minSize));
   };
   return (
-    <FormGroup
-      label="Persistent storage size"
-      helperText={
-        currentSize
-          ? 'Storage size can only be increased. If you do so, the workbench will restart and be unavailable for a period of time that is usually proportional to the size change.'
-          : ''
-      }
-      helperTextIcon={<ExclamationTriangleIcon />}
-      validated={currentSize ? 'warning' : 'default'}
-      fieldId={fieldID}
-    >
+    <FormGroup label="Persistent storage size" fieldId={fieldID}>
       <InputGroup>
-        <NumberInput
-          inputAriaLabel="Persistent storage size number input"
-          id={fieldID}
-          name={fieldID}
-          value={size}
-          min={minSize}
-          onPlus={() => onStep(1)}
-          onMinus={() => onStep(-1)}
-          onChange={(event) => {
-            if (isHTMLInputElement(event.target)) {
-              const newSize = Number(event.target.value);
-              setSize(isNaN(newSize) ? size : newSize);
-            }
-          }}
-          onBlur={(event) => {
-            if (isHTMLInputElement(event.target)) {
-              const blurSize = Number(event.target.value);
-              setSize(Math.max(blurSize, minSize));
-            }
-          }}
-        />
-        <InputGroupText variant="plain">GiB</InputGroupText>
+        <InputGroupItem>
+          <NumberInput
+            validated={currentSize ? 'warning' : 'default'}
+            inputAriaLabel="Persistent storage size number input"
+            id={fieldID}
+            name={fieldID}
+            value={size}
+            min={minSize}
+            onPlus={() => onStep(1)}
+            onMinus={() => onStep(-1)}
+            onChange={(event) => {
+              if (isHTMLInputElement(event.target)) {
+                const newSize = Number(event.target.value);
+                setSize(isNaN(newSize) ? size : newSize);
+              }
+            }}
+            onBlur={(event) => {
+              if (isHTMLInputElement(event.target)) {
+                const blurSize = Number(event.target.value);
+                setSize(Math.max(blurSize, minSize));
+              }
+            }}
+          />
+        </InputGroupItem>
+        <InputGroupText>GiB</InputGroupText>
```

Comment:
Need `isPlain` here too, though the ds-projects feature branch changes might get rid of this soon.